flepimop/gempyor_pkg/src/gempyor/steps_rk4.py

```diff
@@ -134,7 +134,7 @@ def rhs(t, x, today):
                         rate_change_compartment *= parameters[transitions[transition_rate_col][transition_index]][
                             today
                         ][visiting_subpop]
-                        total_rate[spatial_node] *= rate_keep_compartment + rate_change_compartment.sum()
+                        total_rate[spatial_node] *= (rate_keep_compartment + rate_change_compartment.sum())
```

Comment:
Was this producing unexpected results? I think these parens aren't needed.

Reply:
There are there for easier understanding if you don't know priority in pyhton